alacritty_terminal/src/term/mod.rs

```diff
@@ -2219,6 +2225,18 @@ impl<T: EventListener> Handler for Term<T> {
             self.set_title(popped);
         }
     }
+
+    #[inline]
+    fn text_area_size_pixels<W: io::Write>(&mut self, writer: &mut W) {
+        let width = self.size.cell_width * self.cols().0 as f32;
+        let height = self.size.cell_height * self.screen_lines().0 as f32;
```

Comment:
I think printing these as f32 here is potentially dangerous, since it would mangle the escape if these would actually be printed as floating point numbers. So the cell_width and cell_height should probably just be converted to usize.

```suggestion
        let width = self.size.cell_width as usize * self.cols().0;
        let height = self.size.cell_height as usize * self.screen_lines().0;
```